### ISSUE TITLE
[Bug] t.rename: Error SQLDatabaseInterface.update() got an unexpected keyword argument 'mapset'

### ISSUE THREAD

**Issue by veroandreo** [OSGeo/grass#2848]
When trying to rename a strds I get the following:
```
Traceback (most recent call last):
  File "/home/veroandreo/software/grass-dev/dist.x86_64-pc-linux-gnu/scripts/t.rename", line 122, in <module>
    main()
  File "/home/veroandreo/software/grass-dev/dist.x86_64-pc-linux-gnu/scripts/t.rename", line 116, in main
    stds.rename(ident=new_id, dbif=dbif)
  File "/home/veroandreo/software/grass-dev/dist.x86_64-pc-linux-gnu/etc/python/grass/temporal/abstract_space_time_dataset.py", line 2274, in rename
    map.add_stds_to_register(stds_id=ident, dbif=dbif)
  File "/home/veroandreo/software/grass-dev/dist.x86_64-pc-linux-gnu/etc/python/grass/temporal/abstract_map_dataset.py", line 1094, in add_stds_to_register
    self.stds_register.update(dbif=dbif, mapset=mapset)
TypeError: SQLDatabaseInterface.update() got an unexpected keyword argument 'mapset'
```

**To Reproduce**
Run the following within any mapset (set a small region first, just in case):
```
for i in `seq 0 11` ; do
    r.mapcalc expression="map_${i} = ${i}"
    echo map_${i} >> map_list.txt
done
t.create type=strds temporaltype=absolute output=temperature_weekly title="Weekly Temperature" description="Test dataset with weekly temperature"
t.register -i type=raster input=temperature_weekly file=map_list.txt start="2021-05-01" increment="1 weeks"
t.rename input=temperature_weekly output=temp_by_week
```

**System description:**
```
GRASS version: 8.3.dev                                                          
Code revision: b06c460733                                                       
Build date: 2023-02-21                                                          
Build platform: x86_64-pc-linux-gnu                                             
GDAL: 3.4.3                                                                     
PROJ: 8.2.1                                                                     
GEOS: 3.10.2                                                                    
SQLite: 3.36.0                                                                  
Python: 3.10.9                                                                  
wxPython: 4.0.7                                                                 
Platform: Linux-6.1.8-100.fc36.x86_64-x86_64-with-glibc2.35                     
```

**Comment by ninsbl** [comment]
Seems 8.2.1 is affected too...

**Comment by ninsbl** [comment]
Fixed with #2900 , or is further backporting needed (8.0, 7.8)?
